script/Arcueid/Day 6/06_00_ARC06_2_1.txt

```diff
@@ -227,12 +227,13 @@ It's a poor excuse, but there's not really any other way I can explain it.
 -- Page 191, Offset 14376.
 -- 真剣な琥珀さんに対して、“なんでもない”と誤魔化す事はできなかった。
 // 真剣 -> sharpened blade -> too sharp / incisive
+// to be rechecked
```

Comment:
I see 真剣 translated as serious/earnest. Someone also threw out 'overbearing' in chat, but none of these seem to fit well with the sentence imo.

Reply:
Petrikow brought this up as being mistranslated because the current TL is making it sound like any lie Shiki tells, Kohaku would see right through. It means more like "I can't brush off/deceive such an earnest person"